drivers/enc28j60/enc28j60.c

```diff
@@ -248,10 +248,27 @@ static int nd_send(netdev_t *netdev, const iolist_t *iolist)
 
     mutex_lock(&dev->devlock);
 
-#ifdef MODULE_NETSTATS_L2
-    netdev->stats.tx_bytes += count;
-#endif
+    if (cmd_rcr(dev, REG_ECON1, -1) & ECON1_TXRTS)
+    {
```

Comment:
please adapt to RIOT coding style, see [wiki](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions)

Reply:
I guess you would like to have the opening curly brace at the end of the line according to the old ANSI C coding style :-( Personally, I prefere
```
if (condition) 
{
   do_something;
}
else
{
  do_something_else;
}
```
over 
```
if (condition)  {
   do_something;
} else {
  do_something_else;
}
```
because the structure seems to be more clear and much better readable from my point of view, especially in larger code.

Reply:
to be blunt: that's why RIOT (as every other larger project) has coding conventions for consistent style and to avoid re-occuring discussions on that matter.